apps/mobile/src/state/chat/domain.ts

```diff
@@ -70,3 +70,14 @@ export const ChatIds = z.object({
   inboxKey: PublicKeyPemBase64,
 })
 export type ChatIds = z.TypeOf<typeof ChatIds>
+
+export const RequestState = z.enum([
+  'initial',
+  'requested',
+  'denied',
+  'cancelled',
+  'accepted',
+  'deleted',
```

Comment:
Only I can delete the chat. Deleted is if I delete chat - contracts to `otherSideLeft` which means the other side has deleted the chat. I hope this it make sense.